php/laravel/app/Http/Kernel.php

```diff
@@ -14,11 +14,11 @@ class Kernel extends HttpKernel
      * @var array
      */
     protected $middleware = [
-        \App\Http\Middleware\TrustProxies::class,
-        \App\Http\Middleware\CheckForMaintenanceMode::class,
-        \Illuminate\Foundation\Http\Middleware\ValidatePostSize::class,
-        \App\Http\Middleware\TrimStrings::class,
-        \Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull::class,
+//        \App\Http\Middleware\TrustProxies::class,
+//        \App\Http\Middleware\CheckForMaintenanceMode::class,
+//        \Illuminate\Foundation\Http\Middleware\ValidatePostSize::class,
+//        \App\Http\Middleware\TrimStrings::class,
+//        \Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull::class,
```

Comment:
I think it's better to remove this instead of commenting (to keep code as minimal as possible)

Reply:
We could, I left it to make the changes from laravel/laravel more visible.

Reply:
and I'm glad you've taken this path. you're right lets take implementation to minimal in an other `PR`